src/schema/TransferSummary/Query.ts

```diff
@@ -49,12 +49,10 @@ const Query = extendType({
             LEFT JOIN transferFulfilment tF ON t.transferId = tF.transferId
             LEFT JOIN transferParticipant tPPayer ON tPPayer.transferId = t.transferId
                 AND tPPayer.transferParticipantRoleTypeId = (SELECT transferParticipantRoleTypeId from transferParticipantRoleType WHERE name = 'PAYER_DFSP')
```

Comment:
Changed around this query since participantCurrencyId is nullable.
It shouldn't produce any side affects since pCPayer/pCPayer weren't being used for anything other than an unnecessary join.